helix_fhir_client_sdk/graph/fhir_graph_mixin.py

```diff
@@ -21,27 +17,17 @@ async def graph_async(
         id_: str | List[str] | None = None,
         graph_definition: GraphDefinition,
         contained: bool,
-        process_in_pages: Optional[bool] = None,
-        fn_handle_batch: Optional[HandleBatchFunction] = None,
-        fn_handle_error: Optional[HandleErrorFunction] = None,
```

Comment:
why are these removed? does this mean that we don't have to have error and batch handler methods any more??

Reply:
we now use async generators which are better for this